atlasdb-client-protobufs/src/main/proto/TableMetadataPersistence.proto

```diff
@@ -92,6 +92,7 @@ enum ColumnValueFormat {
     PERSISTABLE = 2;
     VALUE_TYPE = 3;
     PERSISTER = 4;
+    REUSABLE_PERSISTER = 5;
```

Comment:
I do not have a great sense of the impact of introducing this. It looks like table metadata is stored to C*, is that right? If yes, what happens on unknown values? Does a node ever try deserializing table metadata from another node?

Reply:
Yeah, in general table metadata is stored in the database. A node **may** deserialize table metadata from another node. Usually this happens if it's an old table that is no longer currently used, *but* for which we may still need to know some metadata about (e.g. for purposes of sweep or other internal tasks) - I can't guarantee that's the only context in which it happens, though.

Protobuf semantics on unknown values are to deserialize to the first constant in the file 🔥 Probably at the time this was written, this kind of extensibility was not thought about.

So this kind of change is a bit tricky. One approach is to do something like what we do for AtlasDB Proxy's Jackson/SMILE persister: we can leave the onus to be on users to ensure they have a schema check-point after they have upgraded their AtlasDB before they move to ReusablePersisters.

Reply:
To be clear, I can also do this without a new entry in the protobuf, this is just what fell out from a simple approach. Given this, let's just serialize it as `PERSISTER`